samples/tab-deeplink/nodejs/views/DeepLink.html

```diff
@@ -0,0 +1,59 @@
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src='https://statics.teams.cdn.office.net/sdk/v1.6.0/js/MicrosoftTeams.min.js'></script>
+        <script src="DeepLinkModel.js"></script>
+     <script>   
+       microsoftTeams.initialize();  
+       $(document).ready(function () {
+       microsoftTeams.getContext(function (context) {          
+ 
+            if (context.subEntityId === "topic1") {              
+                document.getElementById("taskDiv").innerHTML = "Bots :<br/>";
+                document.getElementById("taskContent").innerHTML = "A bot also referred to as a chatbot or conversational bot is an app that runs simple and repetitive automated tasks performed by the users, such as customer service or support staff. Examples of bots in everyday use include, bots that provide information about the weather, make dinner reservations, or provide travel information. A bot interaction can be a quick question and answer, or it can be a complex conversation that provides access to services. For more details <a href='https://docs.microsoft.com/en-us/microsoftteams/platform/bots/what-are-bots' target='_blank'>Click here</a>";
+            }
+
+            else if (context.subEntityId === "topic2") {
+                document.getElementById("taskDiv").innerHTML = "Messaging Extension:<br/>";
+                document.getElementById("taskContent").innerHTML = "Messaging extensions allow the users to interact with your web service through buttons and forms in the Microsoft Teams client. They can search or initiate actions in an external system from the compose message area, the command box, or directly from a message. You can send back the results of that interaction to the Microsoft Teams client in the form of a richly formatted card. This document gives an overview of the messaging extension, tasks performed under different scenarios, working of messaging extension, action and search commands, and link unfurling.For more details <a href='https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions' target='_blank'>Click here</a>";
+            }
+
+            else if (context.subEntityId === "topic3") {
+                document.getElementById("taskDiv").innerHTML = "Adaptive Card :<br/>";
+                document.getElementById("taskContent").innerHTML ="Adaptive cards are a new cross product specification for cards in Microsoft products including Bots, Cortana, Outlook, and Windows. They are the recommended card type for new Teams development. For general information from the Adaptive cards team see Adaptive Cards Overview. You can use adaptive cards anywhere you can use existing Hero cards, Office365 cards, and Thumbnail cards.For more details <a href='https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/what-are-cards' target='_blank'>Click here</a>";
```

Comment:
Instead of constructing html in script, can we hide/show content present in Div?